src/pages/Library.tsx

```diff
@@ -299,12 +342,80 @@ class Library extends React.Component<ILibraryProps, ILibraryState>
                         }
                         
                     </IonGrid>
+                    
                     <div className="next-button-container">
+                    <IonButton fill={"clear"} className="next-button"
+                            onClick={()=>{
+
+                                //THIS SHALL BE THE PREVIOUS BUTTON
+                                console.log("prev < Library.tsx")
+                                console.log(this.state.context.Chapter.statement);
+
+                                //Get current chapter#, which is also SectionNumber.
+                                var curNum = this.state.context.Chapter.statement;
+
+                                //To compensate for a problem from AllPopoverContent <- which sets Chapter.statement to a STRING instead of an INT 
+                                if(typeof curNum === 'string'){
+                                    curNum = parseInt(curNum);
+                                }
+
+                                //This is the next button, so we wanna go to next chapter (+ check edge cases)
+                                if(curNum == 10){
+                                    //Do nothing, at the beginning.
+                                }else if(curNum == 25){
+                                    curNum = 22
+                                }else{
+                                    curNum--;
+                                }
+                                var newContext = {
+                                    Mode: DefaultConfig.typeDisplay,
+                                    Chapter: ({all: 1, saying: 15, statement: curNum}),
+                                    Section: ({all: {Part: 0, SectionNumber: 0}, saying: {Part: 0, SectionNumber: curNum}, statement: {Part: 0, SectionNumber: curNum} }),
+                                    BrowseMode: (DefaultConfig.browseMode)
+                                };
+                                this.setContext(newContext);
+                                   
+                            }}
+                        >
+                            <IonIcon icon = {ellipsisVerticalOutline}></IonIcon>
+                        </IonButton>
                         <IonButton fill={"clear"} className="next-button"
```

Comment:
If this render function gets long and ugly, you can always extract some components -> the components folder. https://reactjs.org/docs/components-and-props.html#extracting-components